### ISSUE TITLE
Ability to resize columns dynamically dragging the middle separator line

### ISSUE THREAD

**Issue by mikhasev** [zamsyt/obsidian-snippets#11]
Subj. Is it possible? Or a separate plugin is required?

**Comment by zamsyt** [comment]
No, I don't think it's possible to have differing widths for the columns with my implementation, and especially not to make a drag interface (using css).
